tests/lowlevel/test_check_format.py

```diff
@@ -0,0 +1,94 @@
+import pytest
```

Comment:
This is a new test that looks at format checking in this cycle / next cycle assignments, making sure they work by default and can be disabled if needed.